engine/security/userns-remap.md

```diff
@@ -245,6 +245,10 @@ for some of these limitations.
 To disable user namespaces for a specific container, add the `--userns=host`
 flag to the `docker container create`, `docker container run`, or `docker container exec` command.
 
+There is a side effect when using this flag: user remapping will not be enabled for that container but, because the read-only (image) layers are shared between containers, ownership of the the containers filesystem will still be remapped.
+
+What this means is that the whole container filesystem will belong to the user specified in the `--userns-remap` daemon config (`231072` in the example above). This can can lean to unexpected bevahior of programs inside the container. For instance `sudo`, which checks that its binaries belong to user `0`.
```

Comment:
A couple minor typos: "This can **lead** to unexpected **behavior**".

Overall the note makes sense and is good information to provide on the current state of userns--setuid binaries are another difficult problem because of this.